.github/wiki/Contribution.md

```diff
@@ -0,0 +1,242 @@
+# Adding a new addon
+
+There are 2 types of addon, Mix addon and Phoenix addon.
+
+Phoenix addon consists of 3 types.
+1. API
+2. Live
+3. Web
+
+![diagram](assets/images/diagram.png)
```

Comment:
is it `diagram` 🤔 should we rename to `addon-structure`, what do you think @topnimble

Reply:
Fixed in a4835fe